xml/operations-troubleshooting-ts_blockstorage.xml

```diff
@@ -690,7 +677,7 @@ Request to delete volume 14b76133-e076-4bd3-b335-fa67e09e51f6 has been accepted.
 2016-04-27 12:39:15.567 6689 DEBUG swiftclient [req-0c65ec42-8f9d-430a-b0d5-05446bf17e34 - - -</screen>
   <para>
    If you determine that the volume is genuinely stuck in
-   <literal>detaching</literal> then you must follow the procedure described in
+   <literal>restoring</literal> then you must follow the procedure described in
```

Comment:
stuck in restoring? Like, a mode? If so, say, "... is stuck in retoring mode, then follow the procedure..."